### DISCUSSION TITLE
Orchard core How to localize the string from the Theme. using PO file

### DISCUSSION THREAD

**Discussion by ziyadsv** [OrchardCMS/OrchardCore#15861] [Q&A]
Am using orchard core and am able to localize the string @T["Prices"] from the pages by creating a folder under App_Data->Sites->Defult->Localisation ar.po and this is working fine but my layout is in TheTheme project, how I will localize the string from theme

**Comment by dodyg** [comment]
The same way. Check this https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

**Comment by agriffard** [comment]
```
msgctxt "TheTheme.Views.Layout"
msgid "Prices"
msgstr "Prices translation"
```

You can use this project to extract your strings to localize: https://github.com/lukaskabrt/PoExtractor

**Comment by ziyadsv** [comment]
If am adding under the folder of Pages or View its showing but if i add new folder its not working.

msgctxt ""ProjectName.Pages.Home.Index""
msgid "Prices"
msgstr "Prices translation"

msgctxt "TheTheme.Views.Shared.Layout"
msgid "Prices"
msgstr "Prices translation"

**Comment by sebastienros** [comment]
Is `TheTheme` your own theme or the one from OC ?

Can you explain step by step what you are doing, with complete filenames, and examples, because right now we are 2 looking at it and don't really understand what `TheTheme` means, which layout you mention, or where your files are.
